Sources/Models/CapabilityOption.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  CapabilityOption.swift
+//  AppStoreConnect-Swift-SDK
+//
+//  Created by Patrick Balestra on 12/22/19.
+//
+
+import Foundation
+
+/// The data structure that represents the resource.
+public struct CapabilityOption: Codable {
+
+    public let description: String?
```

Comment:
Is there a reason to have newlines between every property, maybe project convention?

Reply:
I took a look at the existing convention and there's a newline if there's documentation, but since this type is undocumented by Apple, I can remove the newlines.